Shokofin/LibraryScanner.cs

```diff
@@ -46,6 +48,11 @@ public bool ShouldIgnore(MediaBrowser.Model.IO.FileSystemMetadata fileInfo, Base
                 if (!Lookup.IsEnabledForItem(parent))
                     return false;
 
+                if (fileInfo.IsDirectory &&  Plugin.Instance.IgnoredFolders.Contains(Path.GetFileName(fileInfo.FullName).ToLowerInvariant())) {
+                    Logger.LogDebug("Skipped excluded folder at path {Path}", fileInfo.FullName);
+                    return false;
```

Comment:
Last change; To ignore the directory and all it's sub-entries, return `true` instead of `false` here.

```suggestion
                    return true;
```